packages/lesswrong/components/ea-forum/EAHomeMainContent.tsx

```diff
@@ -129,6 +129,23 @@ type TopicsBarTab = {
   slug?: string
 }
 
+// The order in which the topics are displayed is slightly different from their default ordering
+const topicTabsOrder = [
+  'sWcuTyTB5dP3nas2t', // Global health
+  'QdH9f8TC6G8oGYdgt', // Animal welfare
+  'ee66CtAMYurQreWBH', // Existential risk
+  'ZCihBFp5P64JCvQY6', // Community
+  'oNiQsBHA3i837sySD', // AI safety
+  'H43gvLzBCacxxamPe', // Biosecurity & pandemics
+  '4eyeLKC64Yvznzt6Z', // Philosophy
+  'EHLmbEmJ2Qd5WfwTb', // Building effective altruism
+  'of9xBvR3wpbp6qsZC', // Policy
+  'psBzwdY8ipfCeExJ7', // Cause prioritization
+  'L6NqHZkLc4xZ7YtDr', // Effective giving
+  '4CH9vsvzyk4mSKwyZ', // Career choice
+  'aJnrnnobcBNWRsfAw', // Forecasting & estimation
+]
```

Comment:
Feel free to tell me not to use a list of ids, prob there's a better way. I went with this because I figured this is a special case where we want things in a specific order in one place, and we didn't want changes to the default order to impact this list.